source/help/apps/desktop-changelog.rst

```diff
@@ -1,6 +1,120 @@
 Desktop Application Changelog
 ========================================
 
+Release 4.0.0
+--------------
+
+Release date: January 23, 2018
+
+This release contains multiple security updates for Windows, Mac and Linux, and it is highly recommended that users upgrade to this version.
+
+Improvements
+~~~~~~~~~~~~~~~
+
+All Platforms
+^^^^^^^^^^^^^
+
+- Added a dialog to allow the user to reopen the desktop app if it quits unexpectedly.
+- Mattermost animation icon is now displayed when loading a page, instead of a blank screen.
+- Added a dialog to request permissions to show desktop notifications or to use microphone and video for video calls from untrusted origins.
+- The "Saved" indicator now appears for both Server Management and App Options on the Settings page.
+- Close button on the Settings page now has a hover effect.
+- Added new admin configuration settings for:
+   - Disabling server management where the user cannot add or edit the server URL.
```

Comment:
Needs a space after `Added new admin configuration settings for:` for proper formatting

![image](https://user-images.githubusercontent.com/13119842/35466039-7e297280-02ce-11e8-8e83-8b0659d08101.png)